platform/ui/src/components/StudyListPagination/StudyListPagination.js

```diff
@@ -29,25 +29,17 @@ const StudyListPagination = ({
       <div className="container m-auto relative px-8">
         <div className="flex justify-between">
           <div className="flex items-center">
-            <div className="relative mr-3">
-              <select
-                value={perPage}
-                className="block appearance-none w-full bg-transparent border border-common-active text-white text-base px-2 pr-4 rounded leading-tight focus:outline-none h-8"
-                onChange={e => onChangePerPage(e.target.value)}
```

Comment:
May need to update the `onChange` -- double check.
This was a merge conflict PR that may not have been resolved correctly.